src/trajectory/Serialize.java

```diff
@@ -0,0 +1,28 @@
+package trajectory;
+
```

Comment:
Thanks for the comments Cam. I will try to get the fixes in soon.

Vedaad, I figured out how to use Google GSON with the robot code so I will try to get a basic set up of it pretty soon.